cutlery/_spp.pyx

```diff
@@ -84,6 +84,16 @@ cdef class SentencePieceProcessor:
 
         return pieces
 
+    def piece_to_id(self, str piece):
```

Comment:
The other methods also don't have docs, but we should probably start adding them for new methods?

Reply:
Added docstrings for all methods while I was at it.